app/src/main/res/values/colors.xml

```diff
@@ -185,6 +188,9 @@
     <color name="add_on_private_browsing_exterior_circle_background_private_theme">@color/accent_private_theme</color>
     <color name="add_on_private_browsing_interior_icon_background_private_theme">#FFFFFF</color>
     <color name="prompt_login_edit_text_cursor_color_private_theme">#9059ff</color>
+    <color name="search_suggestion_indicator_icon_color_private_theme">#2ac3a2</color>
```

Comment:
Usually we aim to include 3 resources per color:
- `color_light_theme` which is set to some hex code
- `color_dark_theme` which is set to some other hex code
- `color` which points to the light theme color, then gets overriden to point to the dark theme color inside values-night.

You should be able to just rename the _private colors to _dark_theme, then add an additional three values that can get overriden.

Reply:
I don't think i understand you correctly :)

Reply:
Just the difference between dark theme and private theme is foundation color. So i removed other private color excluding foundation color. Check out latest commit.

Reply:
For example, here are the entries we have for `primaryText`:

```xml
<!-- values/colors.xml -->
<color name="primary_text_light_theme">#20123A</color>
<color name="primary_text_dark_theme">#FBFBFE</color>

<color name="primary_text_normal_theme">@color/primary_text_light_theme</color>
```

```xml
<!-- values-night/colors.xml -->
<color name="primary_text_normal_theme">@color/primary_text_dark_theme</color>
```

`@color/primary_text_normal_theme` just exists to point to light/dark, depending on the current theme.

Reply:
Yes, i did this.